test/offsetPathTest.js

```diff
@@ -35,6 +90,18 @@
         to: 'none'
       });
 
+      assertNoInterpolation({
+        property: 'offset-rotate',
+        from: 'ray(100deg farthest-corner)',
+        to: 'ray(-100deg closest-corner)'
+      });
+
+      assertNoInterpolation({
+        property: 'offset-rotate',
+        from: 'ray(100deg contain farthest-side)',
+        to: 'ray(-100deg closest-corner)'
+      });
+
```

Comment:
Add another assertNoInterpolation without size, i.e. where the change is between having and not having contain.